src/include/esp-idf/bindings.h

```diff
@@ -441,3 +441,20 @@
 #endif // CONFIG_BT_ENABLED
 
 #endif // CONFIG_IDF_TARGET_ESP32S2
+
+//LCD support
+#if defined(CONFIG_IDF_TARGET_ESP32S2) || defined(CONFIG_IDF_TARGET_ESP32S3)
```

Comment:
This does NOT include the original `esp32` chip, not the 'c' variants. Why?

Reply:
See below, neither the original nor the C variants have an LCD interface in their datasheets, as far as I read it.